CafeteriaClient/src/main/java/CafeteriaClient/utils/ConsoleReadUtils.java

```diff
@@ -0,0 +1,96 @@
+package CafeteriaClient.utils;
+
+
+import java.math.BigDecimal;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class ConsoleReadUtils {
+    public static final Scanner scanner = new Scanner(System.in);
+
+    public static int getIntInput(String prompt) {
+        int input = 0;
+        boolean valid = false;
```

Comment:
valid, name is not clear. improve the variable names.
* Check similar things in all places.